cmd/snap/cmd_changes.go

```diff
@@ -122,14 +125,55 @@ func (c *cmdChanges) Execute(args []string) error {
 
 func (c *cmdTasks) Execute([]string) error {
 	cli := Client()
-	return showChange(cli, c.Positional.ID)
+	var id changeID
+	if c.Positional.ID == "" && c.LastChangeType == "" {
```

Comment:
This is a personal preference that may not be shared by the rest of the team but I prefer switch case style instead of the if-then-else tree:

```
switch {
  case c.Positional.ID == "" && c.LastChangeType == "":
  ...
  case c.Positional.ID != "" && c.LastChangeType == "":
  ...
  case c.Positional.ID != "":
  ...
  case c.LastChangeType != "":
  ...
}
```

You could also combine this with my suggestion below and merge the first two cases.